pkg/archive/archive_test.go

```diff
@@ -207,6 +207,164 @@ func TestCmdStreamGood(t *testing.T) {
 	}
 }
 
+func TestUntarPathWithInvalidSrc(t *testing.T) {
+	_, err := os.Create("/tmp/dest")
+	if err != nil {
+		t.Fatalf("Fail to create the destination file")
+	}
+	defer os.RemoveAll("/tmp/dest")
+	err = UntarPath("", "/tmp/dest")
+	if err == nil {
```

Comment:
Its not clear to me what you're testing here. The Fatal() message and name of the test would seem to indicate that you're testing for an invalid src but it could have also failed due to you pointing to a destination file instead of a destination dir.  I would recommend that you split this into two.  One to validate that it generates an error on a bad "src" and one for testing a bad "dest" - but in both cases make sure the other param passed in is valid.